guides/V1_get_team_space_templates.md

```diff
@@ -0,0 +1,39 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'team-space-templates'
+is: 'future'
+---
+
+# Get Team Space Templates
+
+Some space templates are associated with particular teams. 
+Using the teamSpaceTemplates query, a user with the appropriate permission (determined by the offering to which they are subscribed) can fetch a collection of the templates curated by their team admin.
```

Comment:
could you bold the teamSpaceTemplates word to make it stand out?

Reply:
Done.